src/py21cmfast/src/ps.c

```diff
@@ -4292,3 +4370,51 @@ void FreeTsInterpolationTables(struct FlagOptions *flag_options) {
     LOG_DEBUG("Done Freeing interpolation table memory.");
 	interpolation_tables_allocated = false;
 }
+
+double ETHOS_DAOs(double k, struct AstroParams *astro_params_sfrd){
```

Comment:
again, better to have it depend just on `h_PEAK` and `log10_k_PEAK`, then pass those in directly when calling (since it's just two variables) in my opinion.